predicates/methods/methods.go

```diff
@@ -0,0 +1,88 @@
+/*
+
+Package source implements a custom predicate to match routes
```

Comment:
To make godoc happy: `Package source ...` -> `Package methods...`

Reply:
some copy/paste thing 🤦‍♂